common/src/ticker.rs

```diff
@@ -0,0 +1,35 @@
+use crate::Step;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct TickerEntity {
+    pub last_ticked: Step,
+    pub ticker: Ticker,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Ticker {
+    on: bool,
+}
+
+impl Ticker {
+    pub fn new() -> Self {
+        Self { on: false }
+    }
+    pub fn tick(&mut self, _step: Step) {
+        self.on = !self.on;
+
+        // Just for testing
+        if self.on {
+            println!("Ticked ON!");
```

Comment:
Hyprmine uses the `tracing` library for its logging. To apply that here, you can use
```rust
tracing::info!("Ticked ON!");
```

Reply:
Good to know. Changed.